pandas/tests/io/parser/conftest.py

```diff
@@ -59,11 +60,11 @@ def csv1(csv_dir_path):
 
 _py_parsers_only = [_pythonParser]
 _c_parsers_only = [_cParserHighMemory, _cParserLowMemory]
-_all_parsers = _c_parsers_only + _py_parsers_only
+_all_parsers = [*_c_parsers_only, *_py_parsers_only]
```

Comment:
Can you post the error this was causing? The addition seems easer to read so curious why we would need to unpack like this

Reply:
As per the original issue, the error this gave was 
`pandas\tests\io\parser\conftest.py:62: error: Unsupported operand types for + ("List[CParser]" and "List[PythonParser]")`
I was confused if I should just type it as `List[BaseParser]` or the unpacking method. Unpacking is generally the prefered way according to PEP, but less readable as you rightly pointed out. Thinking back, I think the former is a better solution. Your thoughts?

Reply:
Ah makes sense thanks for clarifying and linking PEP. I think this is OK but cc @simonjayhawkins who may have run into this before

Reply:
Sure, @simonjayhawkins thoughts on this?

> As per the original issue, the error this gave was
> `pandas\tests\io\parser\conftest.py:62: error: Unsupported operand types for + ("List[CParser]" and "List[PythonParser]")`
> I was confused if I should just type it as `List[BaseParser]` or the unpacking method. Unpacking is generally the preferred way according to [PEP](https://www.python.org/dev/peps/pep-0448/), but less readable as you rightly pointed out. Thinking back, I think the former is a better solution. Your thoughts?